### PR TITLE
start implementing Reap str skill gem

### PR DESCRIPTION
**Pull request by DavidVollmers**
This is a rudimentary implementation of the Reap str skill gem.
Included is the Physical Dot mod for gems and the Blood Charges Concept.

TODOs:
- add levels 2-20
- str/int requirements
- life cost instead of mana

### CODE REVIEW

File: src/Data/StatDescriptions/gem_stat_descriptions.lua [hunk 1]
```diff
@@ -15885,6 +15885,100 @@ return {
...
+		lang={
+			English={
+				[1]={
+					limit={
+						[1]={
+							[1]="#",
+							[2]="#"
+						}
+					},
+					text="Loose a Blood Charge when an Enemy Dies while affected by this Skill's Debuff"
```
Comment (zuckerjoe) [inline src/Data/StatDescriptions/gem_stat_descriptions.lua:15974]:
```suggestion
					text="Lose a Blood Charge when an Enemy Dies while affected by this Skill's Debuff"
```

File: src/Data/StatDescriptions/gem_stat_descriptions.lua [hunk 2]
```diff
@@ -15885,6 +15885,100 @@ return {
...
+				}
+			}
+		},
+		stats={
+			[1]="loose_blood_charge_on_enemy_death"
```
Comment (zuckerjoe) [inline src/Data/StatDescriptions/gem_stat_descriptions.lua:15979]:
```suggestion
			[1]="lose_blood_charge_on_enemy_death"
```

File: src/Data/StatDescriptions/gem_stat_descriptions.lua (hunk 1 above)
Comment (zuckerjoe) [reply src/Data/StatDescriptions/gem_stat_descriptions.lua:15974]: Hey, don't mean to be petty or anything, just thought I'd let you know there's a typo here while scrolling through. :) Thanks for your work! Keep it up! :)

**Comment by LocalIdentity** [comment]
Fully added in https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/2557

**Pull request closed without merging**